apisix/admin/utils.lua

```diff
@@ -33,8 +33,15 @@ local function inject_timestamp(conf, prev_conf, patch_conf)
         end
     end
 
-    -- For PATCH request, the modification is passed as 'patch_conf'
-    if not conf.update_time or (patch_conf and patch_conf.update_time == nil) then
+    if not conf.update_time or
+        -- For PATCH request, the modification is passed as 'patch_conf'
+        -- If the sub path is used, the 'patch_conf' will be a placeholder `true`
```

Comment:
I think the place holder is nil

Reply:
if the path_conf placeholder is true, 
 (patch_conf and (patch_conf == true or patch_conf.update_time == nil))  =>
(     true and (true == true or true.update_time == nil)   ), true.update_time will raise error

Reply:
sorry, if path_conf is true, true.update_time doesn't executed.